index.html

```diff
@@ -20,25 +20,23 @@
 					<img src="images/welcome/Group-3.png" alt="group">
 				</div>
 			</div>
+		</div>
+	
+		
+		<div class="headline">
 			<nav class="toolbar">
 				<a class="logo left" href="#">My logo</a>
```

Comment:
Kindly work on these changes:
- Add your name as a text instead "logo text".

Reply:
Hi @carlos-ssh, please can you check below if it is what you are requesting:
```suggestion
				<a class="logo left" href="#">Mehdi RAHAL</a>
```